docs/community/developer_guide.mdx

```diff
@@ -83,17 +83,32 @@ To update or add new dependencies, run `go get <package name>`.
 This section needs a lot of love and automation 🙂
 
 :::
-
+### Prerequisites
 1. Make sure your GPG is available on GitHub at `https://github.com/<username>.gpg`. This can be added at https://github.com/settings/keys.
-1. If you haven't already, open PR to add your GPG key to the [`KEYS`](https://github.com/oras-project/oras/blob/main/KEYS) file (see file for instructions).
-1. Open a release PR to
-   - Build with the latest golang: replace go version of the [binary](https://github.com/oras-project/oras/blob/main/.github/workflows/release-github.yml#L32) and [image](https://github.com/oras-project/oras/blob/main/Dockerfile#L14) to match the latest stable version.
-   - Update oras version: replace the [current stable version](https://github.com/oras-project/oras/blob/main/internal/version/version.go#L20) with the upcoming release version.
-1. After the release PR gets merged, [create an issue](https://github.com/oras-project/oras/issues/new) to call for a vote cutting off a release branch named `release-<major>.<minor>` based on the version update commit.
-1. Make a fresh clone of the repo after all above steps are completed. Create a new tag for the version prefixed with "v" and push the tag directly to the repo.
+  -  If you haven't already, open PR to add your GPG key to the [`KEYS`](https://github.com/oras-project/oras/blob/main/KEYS) file (see file for instructions).
+2. Open a bump PR to build with the latest golang: replace go version of the [binary](https://github.com/oras-project/oras/blob/main/.github/workflows/release-github.yml#L32) and [image](https://github.com/oras-project/oras/blob/main/Dockerfile#L14) to match the latest stable version.
```

Comment:
```suggestion
1. Open a bump PR to build with the latest golang: replace go version of the [binary](https://github.com/oras-project/oras/blob/main/.github/workflows/release-github.yml#L32) and [image](https://github.com/oras-project/oras/blob/main/Dockerfile#L14) to match the latest stable version.
```
These should automatically number when rendered

Reply:
The auto-numbering fails when there are bullet points, so I numbered those manually.